### ISSUE TITLE
Swift 5.9?

### ISSUE THREAD

**Issue by FullQueueDeveloper** [swift-actions/setup-swift#575]
**Is your feature request related to a problem? Please describe.**
An error message suggested I post here

```github-actions
Error: Unexpected error, unable to continue. Please report at https://github.com/swift-actions/setup-swift/issues
Version "5.9" is not available
Stacktrace:
Error: Version "5.9" is not available
    at Object.verify (/home/runner/work/_actions/swift-actions/setup-swift/v1/dist/index.js:646:15)
    at /home/runner/work/_actions/swift-actions/setup-swift/v1/dist/index.js:397:36
    at Generator.next (<anonymous>)
    at fulfilled (/home/runner/work/_actions/swift-actions/setup-swift/v1/dist/index.js:377:58)
```

**Describe the solution you'd like**
I would like Swift 5.9 to successfully setup

<img width="927" alt="Screenshot 2023-06-07 at 5 46 07 PM" src="https://github.com/swift-actions/setup-swift/assets/103295874/13834268-9692-4a57-8610-3136ca04f73a">

**Comment by fwal** [comment]
Fixed in #580

**Comment by fwal** [comment]
Sorry jumped the gun here as 5.9 is not yet generally available and this action currently doesn't support dev releases, will have to revisit this.
